src/modules/schema.js

```diff
@@ -123,6 +125,17 @@ export default class Schema {
    * @returns {Boolean}
    */
   static validateSchema(json) {
+    let jsonSchemaSpec;
+    if (json['$schema']) {
+      if (json['$schema'].startsWith('http://json-schema.org/draft-07/schema')) {
```

Comment:
not really a fan of using startswith/if statements here. if we can ensure url doesnt need startswith, we can do like :

```
const jsonSchemas = {
  'http://json-schema.org/draft-07/schema': JSONSchema07
};

const schemaID = json['$schema'];
if (jsonSchemas[schemaID]) {
  jsonSchemaSpec = jsonSchemas[schemaID];
}
```

Reply:
oh also, `["$schema"]` should be written in dot notion - i.e `json.$schema`

Reply:
I will abstract this in a function elsewhere, did that to illustrate the point.